workflows/dragen-transcriptome-pipeline/3.9.3/dragen-transcriptome-pipeline__3.9.3.cwl

```diff
@@ -338,30 +338,6 @@ steps:
       - id: output_directory
       - id: output_file
     run: ../../../tools/multiqc/1.12.0/multiqc__1.12.0.cwl
-  # Step-8: run somalier
-  somalier_step:
-    label: somalier
-    doc: |
-      Runs the somalier extract function to call the fingerprint on the transcriptome bam file
-    in:
-      bam_sorted:
-        # The bam from the dragen transcriptome workflow
-        source: run_dragen_transcriptome_step/dragen_bam_out
-      sites:
-        # The VCF output file from the dragen command
-        source: sites_somalier
```

Comment:
Has sites_somalier been removed from the inputs?

Reply:
updated in https://github.com/umccr/cwl-ica/commit/e746073bd07957de09546a58ed3232f5f1909080